teuthology/orchestra/console.py

```diff
@@ -239,7 +239,8 @@ def power_on(self):
         if self.check_power('on'):
             log.info('Power on for {s} completed'.format(s=self.shortname))
         else:
-            log.error('Failed to power on {s}'.format(s=self.shortname))
+            err_msg = 'Failed to power on {s}'.format(s=self.shortname)
+            raise RuntimeError(err_msg)
```

Comment:
Since we all took direction to py3 I wonder why just not do this:
```
else:
    raise RuntimeError(f"Failed to power on {self.shortname}")
```
And if we add an exception throwing, then this should be reflected in the inline docs for the function, like:
```
:raises: :class:`RuntimeError`: failed to power on
```
Also, I didn't investigate the code enough, but do we need the same in case of error in pair function *power_off*.